pkg/api/api.go

```diff
@@ -315,6 +315,11 @@ func (hs *HTTPServer) registerRoutes() {
 			})
 		})
 
+		// Schema
+		apiRoute.Group("/schema", func(schemaRoute routing.RouteRegister) {
+			schemaRoute.Get("/:filename", routing.Wrap(hs.GetDashboardOrPanelJsonSchema))
```

Comment:
I don't think we want to be this open :)  I would explicitly define things as:
```suggestion
			schemaRoute.Get("/dashboard.json", routing.Wrap(hs.GetDashboardOrPanelJsonSchema))
			schemaRoute.Get("/panel/:pluginid.json", routing.Wrap(hs.GetDashboardOrPanelJsonSchema))
```